bin/yaml/android-java.yaml

```diff
@@ -0,0 +1,9 @@
+compilers:
+  android-d8:
+    type: singleFile
+    dir: r8-{name}
+    targets:
+      - name: 8.1.56
+        filename: r8-8.1.56.jar
+        check_file: r8-8.1.56.jar
```

Comment:
Just as we did with baksmali can we have this depend on a version of java, and then test that the jar executes a "--version" or similar? Hopefully the template/example of baksmali (I made some changes) will help here!

Reply:
Done, thanks for the changes!